server/lib/nl/explore/params.py

```diff
@@ -70,9 +70,15 @@ class DCNames(str, Enum):
 class QueryMode(str, Enum):
   # NOTE: This mode is incompatible with LLM detector
   STRICT = 'strict'
-  # This is a special mode to be used for toolformer experiments.
-  # This mode does not detect topics and has a sv score threshold of 0.8.
-  TOOLFORMER = 'toolformer'
+  # These are special modes used for toolformer experiments.
+
+  # The point mode returns exact variable's values and used by RIG.
+  # It does not detect topics and has a higher sv score threshold of 0.8.
+  TOOLFORMER_POINT = 'toolformer_point'
```

Comment:
given these two mode are not only diff by point/table, should they be called TOOLFORMER_RIG and TOOLFORMER_RAG

Reply:
+1

Reply:
good idea